test/rules.js

```diff
@@ -392,6 +392,7 @@ var tests = {
         ]
     ,   'cr-end':  [
             { doc: "metadata/cr-mediacapture-streams.html", config: { status: "CR" }}
+    ,       { doc: "online/CR-JSON-ld.html", config: { status: "CR"}, errors: ["sotd.cr-end"] } //sotd.cr-end.found-not-valid
```

Comment:
It would probably be better to add smaller test documents like in `test/docs/sotd` and test other use cases:
* feedback due date detected < publication date
* feedback due date detected > publication date but only 27 days 
* feedback due date detected > publication date with more than 28 days

It's not useful to add a 13k lines documents to test only one paragraph.